app/views/play/CampaignView.coffee

```diff
@@ -1406,6 +1406,12 @@ module.exports = class CampaignView extends RootView
       prev = level
       if not @campaign.levelIsPractice(level) and not @campaign.levelIsAssessment(level)
         lastNormalLevel = level
+
+      if level.slug == @courseInstance.get('startLockedLevel') # lock level begin from startLockedLevel
```

Comment:
Should this logic be moved up above this line:
https://github.com/codecombat/codecombat/pull/5649/files#diff-c574a7f240b2737048c1d99833196b8fR1396

Then locking the level will change the color?

Also do we need a `lockedByTeacher` variable? Could `level.locked = true` be in the if statement? `lockedByTeacher` is false by default so this could potentially override other logic? Or is that intended?

I think I am confused and concerned what the impact of this change is and what is intended. Especially because this isn't behind a feature flag.